requirements.txt

```diff
@@ -0,0 +1 @@
+-e git+https://github.com/f5devcentral/f5-ctlr-agent.git@1a9cca04a5bdc45af82c31ef9999d0b65be9f2cc#egg=f5-ctlr-agent
```

Comment:
Why f5devcentral instead of f5networks?

Reply:
This was from @f5yacobucci since this is more of an internal project similar to why CCCL is also f5devcentral.